api/design/resource.go

```diff
@@ -156,4 +156,23 @@ var _ = Service("resource", func() {
 		})
 	})
 
+	Method("ByResourceId", func() {
+		Description("Finds a resource information using resource id")
+		Payload(func() {
+			Attribute("resourceId", UInt, "ID of a resource")
+			Required("resourceId")
+		})
+		Result(Resource, func() {
+			View("default")
```

Comment:
Do we need URLs of versions with the resource? Only `version` and `versionId` would be sufficient I guess  ??
@sthaha